pkg/beholder/global.go

```diff
@@ -0,0 +1,64 @@
+package beholder
+
+import (
+	"sync/atomic"
+
+	"go.opentelemetry.io/otel"
+	otellog "go.opentelemetry.io/otel/log"
+	otellogglobal "go.opentelemetry.io/otel/log/global"
+	otelmetric "go.opentelemetry.io/otel/metric"
+	"go.opentelemetry.io/otel/propagation"
+	oteltrace "go.opentelemetry.io/otel/trace"
+)
+
+// Pointer to the global Beholder Client
+var globalClient = defaultClient()
+
+// SetClient sets the global Beholder Client
+func SetClient(client *Client) {
+	globalClient.Store(client)
+}
+
+// Returns the global Beholder Client
+// Its thread-safe and can be used concurrently
+func GetClient() *Client {
+	return globalClient.Load()
+}
+
+func Logger() otellog.Logger {
+	return GetClient().Logger
+}
+
+func Tracer() oteltrace.Tracer {
+	return GetClient().Tracer
+}
+
+func Meter() otelmetric.Meter {
+	return GetClient().Meter
+}
+
+func MessageEmitter() Emitter {
```

Comment:
```suggestion
func Emitter() Emitter {
```

Reply:
Hmmm that is an unfortunately inconsistency. Should they all be aligned as `Get*()`ers?

Reply:
> Should they all be aligned as Get*()ers?

Sure, see https://github.com/smartcontractkit/chainlink-common/pull/694/commits/fa6cec4a1dd1a5d01f48dfc36220bc66086771db